<launch>

    <!--- Sim Time -->
    <param name="/use_sim_time" value="false"/>
    <!--x y z yaw pitch roll or x y z qua-->
    <node pkg="tf" type="static_transform_publisher" name="static_transform_publisher"
          args="0 0 0 0 0 0 /map /slam 10"/>

    <!--- Run Rviz-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lidar_localization)/launch/my.rviz"/>

    <node pkg="lidar_localization" type="dataLoader" name="dataLoader" output="screen">
        <!--文件夹-->
        <param name="path_to_bin" value="/media/qzj/Dataset/slamDataSet/kitti/data_odometry_velodyne/dataset/sequences/"
               type="str"/>
        <param name="path_to_pose" value="/media/qzj/Dataset/slamDataSet/kitti/data_odometry_velodyne/dataset/poses/"
               type="str"/>
        <param name="seq" value="00" type="str"/>
    </node>

    <node pkg="lidar_localization" type="mapping" name="mapping" output="screen">
    </node>

</launch>